01 - Community/Obsidian Roundup/2023-01-21 More LLM Integrations & Sample Notes for Cooking, Workouts, etc..md

```diff
@@ -10,102 +10,102 @@ Obsidian's nominated for a Golden Kitty in Productivity, there's a new RGB theme
 
 ## In The Community
 
-* Obsidian was nominated on ProductHunt for a [Golden Kitty award](https://www.producthunt.com/golden-kitty-awards/productivity) in the Productivity category.
-* [Scholarships for the Linking Your Thinking Workshop](https://www.linkingyourthinking.com/lyt-workshop-10-scholarship) run by Nick Milo are open until January 21 at 11:59 PST. They read every application; expect results to be communicated by email on January 24.
+* Obsidian was nominated on ProductHunt for a [Golden Kitty award](https://www.producthunt.com/golden-kitty-awards/productivity?ref=obsidian-iceberg) in the Productivity category.
```

Comment:
I'm not sure if this belongs here because the referrer isn't the Iceberg. I don't think it matters too much... not a lot of traffic from the Hub... but maybe Eleanor could decide what she wants to do with this.